### PR TITLE
Game instruction for players 

### PR DESCRIPTION
**Pull request by Takhmina175**
The README file includes the following information:

- Instruction for the players on how to play the TicTacToe game
- Brief information about project set up and language used for building the game
- Author information
- Reference for the project requirements and assisting resources

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,26 @@
...
+- Each player has to enter their names one by one (Note!  You can not leave your name empty)
+- Playing borad <br>
+
+  " # | # | # " <br>
+  "-----------" <br>
+  " # | # | # " <br>
+  "-----------" <br>
+  " # | # | # " <br>
+  
+- The board will be displayed on the screen and the first player will be asked for selecting an empty cell on the board by selecting 1 – 9 number (Note: You cannot enter the number above or below the given range or select occupied space)
```
Comment (kevcap) [inline README.md]: ... for <s>selecting</s> <b>select</b> an empty cell on the board by <s>selecting</s> <b>picking/choosing one number between 1 - 9 </b>.

File: README.md [hunk 2]
```diff
@@ -3,16 +3,26 @@
...
+- The first player’s move will be marked as  ‘X’ and the second player’s move will be marked as ‘O’
+- Each play takes a turn to place either an 'X' or 'O' (a marker) on the board
```
Comment (kevcap) [inline README.md]: Each play<b>er</b> takes a turn ....

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey @Takhmina175,  your project is looking really good, however, to accomplish as the actual project, you required to do these changes, listed below:
 
## [REQUIRED CHANGES]
- [ ] We just have some typo errors, should be very quick to correct them, please check my comments below.

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In the real-world, you'll have a lot of change requests for your real clients, it's good to get used to it.</sub>

**Review by jubaan — APPROVED** [review]
# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

#### :clap: :clap: Great job :hammer_and_pick:!! There are no more issues left to be addressed. 

## See you at the top :rocket:!

# :tada: :tada: :tada: :rotating_light: Status: Approved :rotating_light:  :tada: :tada: :tada:

**Pull request merged**